src/pydocstyle/parser.py

```diff
@@ -80,9 +94,16 @@ def _publicity(self):
         return {True: 'public', False: 'private'}[self.is_public]
 
     @property
+    def source_lines(self):
+        if '_source' in self._fields:
```

Comment:
Will `_source` ever be in `_fields` now? Can't you just make this property a proxy for `self.parent.source_lines` now?